x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -10088,32 +10088,18 @@
     "xpack.crossClusterReplication.unfollowLeaderIndex.confirmModal.singleUnfollowDescription": "フォロワーインデックスは標準のインデックスに変換されます。今後クラスター横断レプリケーションには表示されませんが、インデックス管理で管理できます。この操作は元に戻すことができません。",
     "xpack.crossClusterReplication.unfollowLeaderIndex.confirmModal.unfollowMultipleTitle": "{count} 件のリーダーインデックスのフォローを解除しますか？",
     "xpack.crossClusterReplication.unfollowLeaderIndex.confirmModal.unfollowSingleTitle": "「{name}」のリーダーインデックスのフォローを解除しますか？",
-    "xpack.csp.active_frameworks": "アクティブなフレームワーク",
```

Comment:
Instead of deleting all these translations, consider renaming the IDs, but you'll need to test this as I'm not sure how it'll behave

Reply:
those are auto-generated fixes by running `node scripts/i18n_check.js --fix`

Reply:
i think there's less chance of mistakes if we remove these and let the translators do this